gravitee-apim-console-webui/src/portal/customization/api/api-list/portal-api-list.component.html

```diff
@@ -0,0 +1,149 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (form) {
```

Comment:
No need for this if you add in the component.ts : `form!: FormGroup<ApiForm>`

Reply:
Even if I use `form!: FormGroup<ApiForm>` without that `@if(` I still get this error:
<img width="850" alt="Screenshot 2024-09-09 at 21 16 31" src="https://github.com/user-attachments/assets/7311252c-ce8c-4ab0-9b87-2bf3fa7ca630">



So no, I do need it.